common/v2/services/WalletService/web3/web3.ts

```diff
@@ -46,24 +45,18 @@ export const initWeb3Node = async () => {
   web3Added = true;
   createNode(config, network);
   updateSetting({ ...readAllSettings(), node: 'web3' });
-  return lib;
+  return {nodeLib: lib, network};
 };
 
 export const unlockWeb3 = async () => {
   try {
-    const nodeLib = await initWeb3Node();
+    const { nodeLib, network } = await initWeb3Node();
 
     /*await (action: any) => {
       action.type === configNodesSelectedTypes.ConfigNodesSelectedActions.CHANGE_SUCCEEDED &&
         action.payload.nodeId === 'web3'
     }*/
 
-    const web3Node: any | null = await getWeb3Node();
-    if (!web3Node) {
-      throw Error('Web3 node config not found!');
-    }
-    const network = web3Node.network;
-
```

Comment:
Great job at identifying and solving this bug!

Reply:
Thanks! 😄